### PR TITLE
feat: add portal languages to AppBridgeTheme context

### PR DESCRIPTION
**Pull request by mike85**
This PR adds the portal languages to the AppBridgeTheme context.

### CODE REVIEW

File: packages/app-bridge/src/types/Brandportal.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+export type BrandportalApi = {
+    i18n_settings: {
+        languages: BrandportalI18nLanguage[];
+    };
+};
+
+export type BrandportalI18nLanguage = {
+    language: string;
```
Comment (olivereisenhut) [inline packages/app-bridge/src/types/Brandportal.ts:10]: Is this an ISO language code?

File: packages/app-bridge/src/types/Brandportal.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    label: string;
+    default?: boolean;
+    draft?: boolean;
```
Comment (olivereisenhut) [inline packages/app-bridge/src/types/Brandportal.ts:13]: Could those be defaulted to false instead of having them nullable?

Comment (mike85) [reply packages/app-bridge/src/types/Brandportal.ts]: I changed the type accordingly.

File: packages/app-bridge/src/types/Portal.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+export type PortalLanguage = {
+    isoCode: string;
+    name: string;
+    isDefault: boolean;
+    isDraft: boolean;
```
[the remark this answers is not in the document]
Comment (mike85) [reply packages/app-bridge/src/types/Portal.ts:7]: In general, I find it good practice if you can already identify a boolean type via the property name. However I’ll change it to `default` and `draft` as I just realised that we don’t have any booleans with the `is` prefix in the AppBridge.

**Review by olivereisenhut — APPROVED** [review]

File: packages/app-bridge/src/types/Portal.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (SamuelAlev) [reply packages/app-bridge/src/types/Portal.ts]: Then let's keep the `is` prefix 👍

File: packages/app-bridge/src/types/Language.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+     */
+    name: string;
+
+    /**
+     * Indicates whether the language is the default language.
+     */
+    isDefault: boolean;
+
+    /**
+     * Indicates whether the language is in draft status.
```
Comment (SamuelAlev) [inline packages/app-bridge/src/types/Language.ts:20]:
```suggestion
     * Indicates whether the language is in draft state.
```

**Review by SamuelAlev — APPROVED** [review]

**Pull request merged**